bin/start-local.js

```diff
@@ -1,34 +1,54 @@
 // Run `npm start` and listen for 'DeepForge' then start worker
-var spawn = require('child_process').spawn,
-    stdout = '',
-    execJob,
+const spawn = require('child_process').spawn,
     path = require('path'),
-    env = {cwd: path.join(__dirname, '..'), NODE_ENV: process.env.NODE_ENV},
-    workerJob = null,
-    gmeConfig = require(__dirname + '/../config');
+    gmeConfig = require(__dirname + '/../config'),
+    {SHELL, getDeepForgeServerCommand } = require('./deepforge');
 
-// Set the cache to the blob
-if (gmeConfig.blob.type === 'FS') {
-    process.env.DEEPFORGE_WORKER_CACHE = path.resolve(gmeConfig.blob.fsDir + '/wg-content');
-}
 
-// process.env.NODE_ENV = 'local';
-execJob = spawn('node', [
-    path.join(__dirname, '..', 'app.js')
-], env);
-execJob.stdout.pipe(process.stdout);
-execJob.stderr.pipe(process.stderr);
+const startLocal = function(useCondaInServer) {
+    let stdout, workerJob=null;
+    const env = {cwd: path.join(__dirname, '..'), NODE_ENV: process.env.NODE_ENV, PATH: process.env.PATH},
+        deepForgeServerCommand = getDeepForgeServerCommand(useCondaInServer);
+    // Set the cache to the blob
+    if (gmeConfig.blob.type === 'FS') {
+        process.env.DEEPFORGE_WORKER_CACHE = path.resolve(gmeConfig.blob.fsDir + '/wg-content');
+    }
+
+    // process.env.NODE_ENV = 'local';
+    const execJob = spawn(deepForgeServerCommand, [
+        path.join(__dirname, '..', 'app.js')
+    ], {
+        env: env,
+        shell: SHELL
+    });
 
-execJob.stdout.on('data', function(chunk) {
-    if (!workerJob) {
-        stdout += chunk;
-        if (stdout.indexOf('DeepForge') > -1) {
-            workerJob = spawn('npm', ['run', 'worker'], env);
-            workerJob.stdout.pipe(process.stdout);
-            workerJob.stderr.pipe(process.stderr);
-            workerJob.on('close', code => code && process.exit(code));
+    execJob.stdout.pipe(process.stdout);
+    execJob.stderr.pipe(process.stderr);
+
+    execJob.stdout.on('data', function(chunk) {
+        if (!workerJob) {
+            stdout += chunk;
+            if (stdout.indexOf('DeepForge') > -1) {
+                workerJob = spawn('npm', ['run', 'worker'], env);
+                workerJob.stdout.pipe(process.stdout);
+                workerJob.stderr.pipe(process.stderr);
+                workerJob.on('close', code => code && process.exit(code));
+            }
         }
-    }
-});
+    });
 
-execJob.on('close', code => code && process.exit(code));
+    execJob.on('close', code => code && process.exit(code));
+};
+
+if(require.main === module) {
+    let useCondaInServer;
+    if(process.argv.includes('--help')){
+        process.argv.pop();
+    }
+    if(process.argv.length < 3) {
+        useCondaInServer = true;
+    } else {
+        useCondaInServer = process.argv[2] === 'true';
+    }
```

Comment:
This argument parsing is a little problematic. It looks like you are expecting one of three options: no arguments (use conda), with a `--help` flag, or with `true`. However, I don't see why we are removing `--help` - that flag wasn't recognized anyway so it shouldn't cause any problems to leave it. (The logic to remove it is actually a bit problematic as we don't have a guarantee that the `--help` flag is the last argument so we could be popping off some other flag...)

I would suggest either changing these lines to simply 
```
const useCondaInServer = process.argv[2] === 'true';
```
or you could use a different flag for consistency with the actual deepforge cli:
```
const useCondaInServer = !process.argv.includes('--no-conda');
```
However, consistency between the deepforge cli and this script is not a big deal since this script isn't used by end users.

Reply:
This file has been removed as it was used internally and had no `cli` usage, has been wrapped in a function in `./bin/deepforge`.